rafaelpadilla/review_object_detection_metrics
# Docker image and Docker compose file

This PR adds a `Dockerfile` and a `docker-compose` file to run the Review Object detection metrics GUI in any linux environment.  I built the docker and run in my environment. I haven't tested it for Windows.

### How to test

In a terminal,  please run

```shell
xhost +local:
```

then run the following 

```shell
docker compose up
```

The GUI should show up.
